### PR TITLE
VZ-9653: Update required permissions by fluentd in troublehsooting guide

### PR DESCRIPTION
**Pull request by karan56625**
updated required permissions by fluentd in troublehsooting guide

### CODE REVIEW

File: content/en/docs/troubleshooting/troubleshooting-fluentd.md [hunk 1]
```diff
@@ -30,6 +30,12 @@ $ kubectl logs <fluentd-pod-name> -n verrazzano-system
    ```
 3. Resolve permission issue.
     - The issue occurs when SELinux is in `enforcing` mode on the worker nodes and Fluentd does not have the appropriate SELinux context to have read/write access to the logs directory.
+      Fluentd container requires the following permissions:
```
Comment (gaurav-05) [inline content/en/docs/troubleshooting/troubleshooting-fluentd.md:33]: Will it be better to move it below where we are suggesting to create custom policy.

Comment (karan56625) [reply content/en/docs/troubleshooting/troubleshooting-fluentd.md]: moved it there. Let me know if it looks better. Otherwise we can revert it back.

**Comment by gilbode** [comment]
I wonder if this would be more useful if we had a section somewhere about running Verrazzano on clusters where the nodes are in selinux enforcing mode so users could proactively set things up properly in that scenario.  We could still have the troubleshooting section bit it could simply say - are you running selinux in enforcing mode?  if so ensure you set things up as described.

**Comment by karan56625** [comment]
> I wonder if this would be more useful if we had a section somewhere about running Verrazzano on clusters where the nodes are in selinux enforcing mode so users could proactively set things up properly in that scenario. We could still have the troubleshooting section bit it could simply say - are you running selinux in enforcing mode? if so ensure you set things up as described.
> 
> By only putting in troubleshooting we don't tell people how to set things up correctly in the first place and force them to have a problem before telling how to fix it.

The changes have been pushed according to your suggestion. Please let me know if any further modifications are required

**Review by gilbode — APPROVED** [review]
Looks good to me

**Review by MelwynPaul — APPROVED** [review]

**Pull request merged**